New Group/DetailScreen/MovieDetailViewController.swift

```diff
@@ -0,0 +1,64 @@
+import UIKit
+
+class MovieDetailViewController: UIViewController {
+    @IBOutlet private weak var backGroundImageView: UIImageView!
+    @IBOutlet private weak var avatarImage: UIImageView!
+    @IBOutlet private weak var nameMovieLabel: UILabel!
+    @IBOutlet private weak var overviewLabel: UILabel!
+    @IBOutlet private weak var dateLabel: UILabel!
+    @IBOutlet private weak var averageRatingLabel: UILabel!
+    @IBOutlet private weak var rateCountLabel: UILabel!
+    @IBOutlet private weak var popularityLabel: UILabel!
+    
+    var viewModel: MovieDetailViewModel!
+    var coordinator: MovieDetailCoordinator!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        fetchMovieDetail()
+        configViewModel()
+    }
+    
+    @IBAction func didTapPopToHomeScreenButton(_ sender: Any) {
+        navigationController?.popViewController(animated: true)
+    }
+    
+}
+
+
+//MARK: Fetch data
+extension MovieDetailViewController {
```

Comment:
why does this extension has fetching data, and update data? Also, the configViewModel. You should separate those into sections such as: fetch data, configure UI